ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionValidator.java

```diff
@@ -202,11 +203,6 @@ private ValidationResult<TransactionInvalidReason> validateCostAndFee(
       }
     }
 
-    if (transaction.getNonce() == MAX_NONCE) {
```

Comment:
This needs to stay.

Reply:
removed since it is a duplication, the check is present at line 156